storybook/src/components/form/form-group-radios.css-template.ts

```diff
@@ -31,8 +31,9 @@ export const cssFormGroupRadios: ComponentImplementation<FormGroupRadios<Templat
           aria-describedby=${ifDefined(ariaDescribedBy)}
           aria-errormessage=${ifDefined(ariaErrorMessage)}
         >
+          <legend class="sr-only">${formGroup.label}</legend>
           <div class="dso-label-container">
-            <label for=${formGroup.id} class="control-label"> ${formGroup.label} </label>
+            <span class="control-label" aria-hidden="true"> ${formGroup.label} </span>
```

Comment:
De extra spaties hier mogen weg, die staan ook niet in `form-group-checkboxes.css-template.ts`.